### ISSUE TITLE
Compatibility issue with Laravel 10

### ISSUE THREAD

**Issue by diguida-n** [advoor/nova-editor-js#94]
I'm updating my laravel 9 project to use Laravel 10, but I'm encountering an error related to compatibility. I believe there is a conflict between the requirements of package and Laravel 10. Can you please help me resolve this issue?

```
{
    "require": {
        "php": "^8.1.0",
        "advoor/nova-editor-js": "^3.0",
        "laravel/framework": "^10.0"
    }
}
```

```
Problem 1
    - Conclusion: don't install advoor/nova-editor-js v3.0.1 (conflict analysis result)
    - Conclusion: don't install advoor/nova-editor-js v3.0.5 (conflict analysis result)
    - Conclusion: don't install advoor/nova-editor-js v3.1.0 (conflict analysis result)
    - illuminate/support[v8.0.0, ..., v8.11.2] require php ^7.3 -> your php version (8.2.2) does not satisfy that requirement.
    - Root composer.json requires advoor/nova-editor-js ^3.0 -> satisfiable by advoor/nova-editor-js[v3.0.0, ..., v3.1.0].
    - Conclusion: don't install laravel/framework v10.0.1 (conflict analysis result)
    - Conclusion: don't install laravel/framework v10.0.2 (conflict analysis result)
    - Conclusion: don't install laravel/framework v10.0.3 (conflict analysis result)
    - Conclusion: don't install laravel/framework v10.0.0 (conflict analysis result)
    - advoor/nova-editor-js v3.0.0 requires illuminate/support ^8.0 || ^9.0 -> satisfiable by illuminate/support[v8.0.0, ..., 8.x-dev, v9.0.0-beta.1, ..., 9.x-dev].
    - Only one of these can be installed: illuminate/support[v5.0.0, ..., 5.8.x-dev, v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev, v9.0.0-beta.1, ..., 9.x-dev, v10.0.0, ..., 10.x-dev], laravel/framework[v10.0.0, ..., 10.x-dev]. laravel/framework replaces illuminate/support and thus cannot coexist with it.
    - Root composer.json requires laravel/framework ^10.0 -> satisfiable by laravel/framework[v10.0.0, ..., 10.x-dev].
```

**Comment by Woeler** [comment]
I fixed this in https://github.com/advoor/nova-editor-js/pull/93

But waiting for the maintainers to merge and release.

**Comment by desaintflorent** [comment]
Any chance this PR get merged soon ? I'm also waiting to use Laravel 10

**Comment by roelofr** [comment]
Laravel 10 support is available starting in v3.2.0

**Comment by Woeler** [comment]
Cheers mate!
